tests/foreman/sys/test_fam.py

```diff
@@ -216,6 +216,6 @@ def test_positive_run_inventory(module_target_sat, setup_fam, ansible_module):
     """
     # Execute test_playbook
     result = module_target_sat.execute(
-        f'cd {FAM_ROOT_DIR} && NO_COLOR=1 PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 pytest-3.11 tests/test_crud.py::test_inventory[{ansible_module}]'
+        f'cd {FAM_ROOT_DIR} && NO_COLOR=1 PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 pytest-3.12 tests/test_crud.py::test_inventory[{ansible_module}]'
```

Comment:
Just curious to know, how and where we install pytest on sat for FAM tests?

Reply:
There is a `fam-test-setup.yml` playbook in Tower that does that.